twisted/web/_http2.py

```diff
@@ -686,7 +686,18 @@ def openStreamWindow(self, streamID, increment):
         @type increment: L{int}
         """
         # TODO: Consider whether we want some kind of consolidating logic here.
```

Comment:
I think that is better to have a ticket for this todo and convert it into a FIXME

In the ticket we can add more information, notes and discuss this issue